license-management.md

```diff
@@ -0,0 +1,15 @@
+# License Management
+
+Kontena Pharos PRO version requires a valid Kontena Pharos license. However, you can freely evaluate all features for 30 days.
```

Comment:
Maybe mention that user can contact sales@kontena.io if need extended evaluation period?